source/documentation/install-guide/chap-Configuring_Storage.html.md

```diff
@@ -16,6 +16,8 @@ There are Installing_the_Websocket_Proxy_on_a_Separate_Machine types of storage
 
 * **ISO Domain:** ISO domains store ISO files (or logical CDs) used to install and boot operating systems and applications for the virtual machines, and can be shared across different data centers. An ISO domain removes the data center's need for physical media. ISO domains can only be NFS-based. Only one ISO domain can be added to a data center.
 
+    **Important:** ISO domains have been deprecated, ISO images should be uploaded directly on Data Domains from the web admin console.
```

Comment:
I'd just suggest a slight edit: "ISO domains have been deprecated. ISO images should be uploaded directly to data domains from the Administration Portal." (RHV docs only uses "Administration Portal", but feel free to disregard that suggestion if that isn't true for the oVirt docs).

Reply:
thanks for the review, I've updated the message with your suggestion